geode-core/src/main/java/org/apache/geode/internal/cache/persistence/PersistenceAdvisorImpl.java

```diff
@@ -860,19 +895,20 @@ public InitialImageAdvice getInitialImageAdvice(InitialImageAdvice previousAdvic
           }
         }
       }
+      removeOlderMembers(membersToWaitFor);
```

Comment:
I think it would also be safer to also call removeOlderMembers on "offlineMembers". At the beginning of this method we did an "addAll" on it of all the previouslyOnlineMembers so I think offline could also have old ids.

Reply:
After talking to Darrel, this change will go into the branch for GEODE-5413